src/workflow/workflow.js

```diff
@@ -863,6 +825,68 @@ const verifyWorkflow = async (wfConfig) => {
   }
 };
 
+const calcMinInitialFund = async () => {
+  const context = getContext();
+  const { api, signingPair } = context.network;
+
+  let collectionId = 1;
+  let itemId = 1;
+  // calculate minimum initial fund
+  const destinationAddress = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY';
+  const { existentialDeposit } = api.consts.balances;
+  const info = await api.tx.uniques
+    .transfer(collectionId, itemId || 0, destinationAddress)
+    .paymentInfo(signingPair.address);
```

Comment:
Just curious, could these rpc method be used to get the estimate? https://polkadot.js.org/docs/substrate/rpc/#payment

Reply:
This is just a utility wrapper around that rpc. you can also get the encoded call and call the RPC but this looks more ergonomic!